deployment_samples/EdgeDataStore/Install/templates/installation_files/README.md

```diff
@@ -0,0 +1 @@
+Add your installation files here.
```

Comment:
I would probably say `Add your installation files to this folder` (or directory)